owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt

```diff
@@ -155,6 +159,18 @@ open class FolderPickerActivity : FileActivity(),
 
     override fun onCreateOptionsMenu(menu: Menu?): Boolean {
         menuInflater.inflate(R.menu.main_menu, menu)
+        menu?.let {
+            it.findItem(R.id.action_search)?.let { menuItem ->
```

Comment:
In this case, we don't need a nullable check for `findItem` since it returns `MenuItem!`, or which is the same, a non-nullable `MenuItem`

Reply:
Ok, I have deleted that nullable check and it's working.  👍